src/settings/LocationLocations/RemoteStorageDetails.js

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {
+  Col,
+  KeyValue,
+  Row,
+} from '@folio/stripes/components';
+
+import { useRemoteStorageApi } from './RemoteStorage';
+
+const RemoteStorageDetails = ({ locationId }) => {
+  const { remoteMap, configurations } = useRemoteStorageApi();
+
+  const currentConfig = useMemo(() => {
+    return configurations?.records.find(config => remoteMap[locationId] === config.id);
+  }, [locationId, remoteMap]);
+
+  return (
+    <>
+      {currentConfig?.name && (
+        <Row>
+          <Col xs={12}>
+            <KeyValue
+              label={<FormattedMessage id="ui-tenant-settings.settings.location.locations.remoteStorage" />}
+              value={currentConfig?.name}
+            />
+          </Col>
+        </Row>
+      )}
+
+      {currentConfig?.returningWorkflowDetails && (
+        <Row>
+          <Col xs={12}>
+            <KeyValue
+              label={<FormattedMessage id="ui-tenant-settings.settings.location.locations.returning-workflow.title" />}
+              value={
+                <FormattedMessage
+                  id={`ui-tenant-settings.settings.location.locations.returning-workflow.${currentConfig?.returningWorkflowDetails}`}
```

Comment:
We should probably handle the case of `currentConfig?.returningWorkflowDetails` being undefined